__tests__/app.test.js

```diff
@@ -112,6 +112,101 @@ describe("/api/articles/:article_id", () => {
                 expect(msg).toBe("invalid article id");
             });
     });
+
+    // PATCH requests
+    test("PATCH - 200: returns updated article when passed an object with a inc_votes property", () => {
+        return request(app)
+            .patch("/api/articles/1")
+            .send({ inc_votes: 20 })
+            .expect(200)
+            .then(({ body: { article } }) => {
+                expect(article).toEqual({
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
+                    created_at: expect.any(String),
+                    votes: 120
+                });
+            });
+    });
+    test("PATCH - 200: returns as normal, ignoring superfluous properties", () => {
+        return request(app)
+            .patch("/api/articles/1")
+            .send({
+                inc_votes: 20,
+                topic: "hello",
+                actorWhoPlaysDrEvilAustinPowersAndFatBastardInTheRenownedAndExcellentAustinPowersTrilogy:
+                    "Mike Myers"
+            })
+            .expect(200)
+            .then(({ body: { article } }) => {
+                expect(article).toEqual({
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
+                    created_at: expect.any(String),
+                    votes: 120
+                });
+            });
+    });
+    test("PATCH - 200: sets votes property to 0 if passed object makes it negative", () => {
```

Comment:
This is optional based on your desired functionality. Presumably, you don't want the front-end vote count to ever be negative for this article.

Reply:
That's correct. I figured a negative vote count doesn't make sense. It's a tad inefficient, though, because it requires two db queries...